### DISCUSSION TITLE
A new theme

### DISCUSSION THREAD

**Discussion by rollecode** [stashapp/stash#3884] [Show and tell]
Hello. I have a theme someone could like. Feel free to use it any way you want or implement it [here](https://github.com/stashapp/stash/wiki/Themes).

https://p.fr33tux.org/512957

**Comment by DogmaDragon** [comment]
Thanks. Posted https://github.com/stashapp/CommunityScripts/tree/main/themes/Theme-PornHub

**Reply by rollecode** [reply]
@DogmaDragon Thank you for this. Just noticed in the plugin list. Can you obfuscate my name, I'd rather not use my full real name there in the list :) You can use an anagram neurokinin for example. Also, here's [the newest version](https://p.fr33tux.org/3269a7). I'm posting this in this discussion because not very comfortable in contributing this repo-wise and thougth someone might like this so this is the way...

**Reply by DogmaDragon** [reply]
Thanks, updated and author credit changed. https://github.com/stashapp/CommunityScripts/pull/384
